### DISCUSSION TITLE
Why does not the default snapshot strategy remove the old and useless dump file?

### DISCUSSION THREAD

**Discussion by XiaooLei** [dragonflydb/dragonfly#1353] [Q&A]
I am trying to use dragonfly in our systems, but when I was configuring it, I found the the default strategy just don't remove the old snapshot files even if new snapshot file is generated, and as time goes, the snapshot files grows so big and takes up so much storage space. this is different than the Redis snapshot strategy. I finally managed to make it align to Redis by adding command args '''--dbfilename dump.rdb --nodf_snapshot_format''' , But I still can not  figure out why dragonfly team use such a seeming useless snapshot strategy? Really appreciated if my question could be answered

**Comment by romange** [comment]
@XiaooLei why did you specify `--nodf_snapshot_format`?

**Reply by liusf1993** [reply]
I didn't find any doc abount flag `nodf_snapshot_format`, and only found flag `df_snapshot_format`, 
> df_snapshot_format - Set true to save dump in Dragonfly file format (true by default).

Is it be the  same  with `df_snapshot_format=false` ?
It seems that while `df_snapshot_format=true` , snapshot dump file name has timestamps, so many snapshots may kept.
While `df_snapshot_format=false', it is the same with redis snapshot strategy.Is my conjecture correct?

**Reply by liusf1993** [reply]
Oh,I misunderstood, I saw the explaination in the link`https://github.com/dragonflydb/dragonfly/issues/1377`
